dls-controls/malcolmjs
# Feature/configurable footer #79

## Description

Updated a few of the top level components so they have a configurable height

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Update your settings.json with a `footerHeight: 45` property, start the site and note the white space at the bottom. 
- If you want you can copy the code from `panda.fotter.html` into `index.html` and see what the panda footer will look like on the deployed system.

## Agile board tracking

connect to #79 
closes #79
